crates/ruma-client-api/src/r0/membership/invite_user.rs

```diff
@@ -32,6 +32,12 @@ ruma_api! {
         /// The user to invite.
         #[ruma_api(body)]
```

Comment:
This should fix the error from `ruma_api!`.

```suggestion
        #[serde(flatten)]
```